dashboard/client/src/pages/overview/cards/RecentJobsCard.tsx

```diff
@@ -39,89 +24,27 @@ export const RecentJobsCard = ({ className }: RecentJobsCardProps) => {
 
   const sortedJobs = _.orderBy(jobList, ["startTime"], ["desc"]).slice(0, 6);
 
-  return (
-    <OverviewCard className={classNames(classes.root, className)}>
-      <Typography variant="h3">Recent jobs</Typography>
-      <div className={classes.listContainer}>
-        {sortedJobs.map((job) => (
-          <RecentJobListItem
-            key={job.job_id ?? job.submission_id}
-            className={classes.listItem}
-            job={job}
-          />
-        ))}
-        {sortedJobs.length === 0 && (
-          <Typography variant="h4">No jobs yet...</Typography>
-        )}
-      </div>
-      <LinkWithArrow text="View all jobs" to="/jobs" />
-    </OverviewCard>
-  );
-};
-
-const useRecentJobListItemStyles = makeStyles((theme) =>
-  createStyles({
-    root: {
-      display: "flex",
-      flexDirection: "row",
-      flexWrap: "nowrap",
-      alignItems: "center",
-      textDecoration: "none",
-    },
-    textContainer: {
-      flex: "1 1 auto",
-      width: `calc(100% - ${theme.spacing(1) + 20}px)`,
-    },
-    title: {
-      color: "#036DCF",
-    },
-    entrypoint: {
-      overflow: "hidden",
-      textOverflow: "ellipsis",
-      whiteSpace: "nowrap",
-      color: "#5F6469",
-    },
-    icon: {
-      marginRight: theme.spacing(1),
-    },
-  }),
-);
-
-type RecentJobListItemProps = {
-  job: UnifiedJob;
-  className?: string;
-};
-
-const RecentJobListItem = ({ job, className }: RecentJobListItemProps) => {
-  const classes = useRecentJobListItemStyles();
-
-  const cardContent = (
-    <React.Fragment>
-      <JobStatusIcon className={classes.icon} job={job} />
-      <div className={classes.textContainer}>
-        <Typography className={classes.title} variant="body2">
-          {job.job_id ?? job.submission_id}
-        </Typography>
-        <Typography
-          className={classes.entrypoint}
-          title={job.entrypoint}
-          variant="caption"
-        >
-          {job.entrypoint}
-        </Typography>
-      </div>
-    </React.Fragment>
-  );
+  const sortedJobToRender = sortedJobs.map((job) => {
+    return {
+      title: job.job_id ?? job.submission_id ?? undefined,
+      subtitle: job.entrypoint,
+      link:
+        job.job_id !== null && job.job_id !== ""
+          ? `?/jobs/${job.job_id}`
+          : undefined,
+      className: className,
+      icon: <JobStatusIcon className={classes.icon} job={job} />,
+    };
+  });
 
   return (
-    <div className={className}>
-      {job.job_id !== null && job.job_id !== "" ? (
-        <Link className={classes.root} to={`/jobs/${job.job_id}`}>
-          {cardContent}
-        </Link>
-      ) : (
-        <div className={classes.root}>{cardContent}</div>
-      )}
-    </div>
+    <ListItemCard
+      headerTitle="Recent jobs"
+      className={className}
+      items={sortedJobToRender}
+      emptyListText="No jobs yet..."
+      footerText="View all jobs"
+      footerLink="/jobs"
+    ></ListItemCard>
```

Comment:
```suggestion
    />
```
nit

Reply:
Ops!I forgot this rule.How about we add eslint rule "self-closing-comp" to dashboard UI? I could do it in a separate PR